fake-cucumber/javascript/src/cli.ts

```diff
@@ -8,21 +8,27 @@ import { version } from '../package.json'
 import * as messages from '@cucumber/messages'
 import createMeta from '@cucumber/create-meta'
 import { MessageToNdjsonStream } from '@cucumber/message-streams'
+import { RunOptions } from './types'
 
 const program = new Command()
 program.version(packageJson.version)
 program.option('-r, --require <path>', 'override require path')
+program.option('--retry <count>', 'allow up to <count> retries')
```

Comment:
Can we make it explicit that it's retries of *each scenario* here?

Reply:
Updated